### PR TITLE
added tests, getCard, abstraction of Game

### PR DESCRIPTION
**Pull request by IStallcup**

### CODE REVIEW

**Comment by IStallcup** [comment]
Most recent PR removes abstract nature of game as I can't write tests for the abstract class without a way to instantiate it, but I'll make sure to make it easy to change the code to factor in the abstract game class.

File: src/main/java/models/Game.java [hunk 1]
```diff
@@ -37,4 +40,21 @@ public Game()
...
+	//will return the card it chooses
+	public /*Card*/ Integer getCard()
+	{
+		int removedIdx = rnd.nextInt(deck.size());
+		/*Card*/ Integer removedCard = deck.get(removedIdx);
+		deck.remove(removedIdx);
+		return removedCard;
+	}
+
+	public void hit()
```
Comment (athai) [inline src/main/java/models/Game.java]: All this function does is call getCard(), so it doesn't feel necessary.

Comment (IStallcup) [inline src/main/java/models/Game.java]: Good point, I'll rearrange. :+1:

File: src/test/java/models/gameTest.java [hunk 2]
```diff
@@ -14,6 +14,27 @@ public void testGameStart()
...
+        assert(g.deck.size() == 51);
+    }
+
+    @Test
+    public void testDealTwo()
+    {
+        Game g = new Game();
+        g.getCard();
+        g.getCard();
+        assert(g.hand.size() == 2);
```
Comment (athai) [inline src/test/java/models/gameTest.java]: May want to use assertEquals() instead

File: src/test/java/models/gameTest.java [hunk 3]
```diff
@@ -1,7 +1,7 @@
 package models;
 
 import org.junit.Test;
-import org.junit.Assert.*;
+import org.junit.Assert;
```
Comment (athai) [inline src/test/java/models/gameTest.java]: import static org.junit.Assert.*

**Comment by IStallcup** [comment]
Changes made! :+1: ?

File: .travis.yml [hunk 4]
```diff
@@ -3,7 +3,6 @@ sudo: false
 language: java
 
 jdk:
-  - oraclejdk7
```
Comment (leel8on) [inline .travis.yml:6]: Thanks for removing.

**Comment by leel8on** [comment]
Looks good. :+1:

File: src/test/java/models/gameTest.java [hunk 5]
```diff
@@ -14,6 +15,27 @@ public void testGameStart()
...
+        assertEquals(g.deck.size(), 52);
     }
+
+    @Test
+    public void testGetCardDecrementsSize()
+    {
+        Game g = new Game();
+        //when Game is abstract, replace with instatiation of actual class
+        /*Card*/ Integer card1 = g.getCard();
+        assertEquals(g.deck.size(), 51);
```
Comment (kannabanana) [inline src/test/java/models/gameTest.java:28]: Don't magic number

**Pull request merged**